src/components/AutocompleteLocation.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+    <div class="columns">
+    <GmapPlaceInput
```

Comment:
If so: could you tell me where GmapPlaceInput component is explicitly mounted in this (AutocompleteLocation) component? ;)